dumpling-overview.md

```diff
@@ -248,7 +248,8 @@ Dumpling 也可以通过 `-B` 或 `-T` 选项导出特定的数据库/数据表
 
 默认情况下，导出的文件会存储到 `./export-<current local time>` 目录下。常用选项如下：
 
-- `-t` 用于指定导出的线程数。增加线程数会增加 Dumpling 并发度，但也会加大数据库内存消耗，因此不宜设置过大。
+- `-o` 用于选择存放导出文件的目录。
```

Comment:
Please align the latest version of this doc: https://github.com/pingcap/docs/blob/master/dumpling-overview.md#improve-export-efficiency-through-concurrency

Reply:
Updated~